addons/html_editor/static/src/others/qweb_plugin.js

```diff
@@ -12,6 +12,7 @@ export class QWebPlugin extends Plugin {
     });
 
     setup() {
+        this.editable.classList.add("odoo-editor-qweb");
```

Comment:
If you do that and @ged-odoo agree, you should remove
```js
    get classList() {
        return this.config.pluginSet === "extras" ? ["odoo-editor-qweb"] : [];
    }
```
as it become unnecessary

Reply:
ahhh that why it didn't work on my plugins set.
I think my solution is more robust.

Reply:
we should probably remove it on destroy ?